MetaMask/metamask-extension
# feat(migration): enable NFT auto-detection when basic functionality is on

## **Description**

Adds a migration that enables NFT auto-detection if the basic functionality toggle is on

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26406?quickstart=1)

## **Related issues**

## **Manual testing steps**

- Install a previous build
- In settings, disable NFT auto detection but keep basic functionality on
- Upgrade to this build, reload extension
    - (background console should say `Running migration 127...`)
- Verify NFT autodetection is enabled in settings

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
